docs/docs/user-guide/using-cli-vs-code-extension.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Use CLI for Microsoft 365 VS Code extension
+sidebar_position: 13
+---
+
+# Use CLI for Microsoft 365 VS Code extension
+
+If you like using Visual Studio Code to write scripts as your day-to-day tool then you might want to try the CLI for Microsoft 365 VS Code extension to boost your productivity. 
+It provides various features that may be helpful when creating scripts using CLI for Microsoft 365.
+
+You may download the VS Code extension directly from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=adamwojcikit.cli-for-microsoft-365-extension) or install it from the VS Code extension gallery.
+
+Currently, the extension provides the following:
+
+## Full docs for every CLI for Microsoft 365 command
+
+The extension allows you to browse CLI for Microsoft 365 docs about every command inside VS Code. 
+
+![Docs view](../images/cli-vs-code-extension/Docs.png)
+
+No more transition between the code editor and browser is needed. 
+It is possible to go to docs original website location and also open a sample gallery showing all samples using the command.
+
+## Samples gallery with samples
```

Comment:
If I'm not mistaken, it's `sample gallery` (singular). Also `with samples` is quite obvious I think 😊

```suggestion
## Sample gallery
```